### PR TITLE
[CELEBORN-1794] Fix TestCongestionController occasional failing

### PR DESCRIPTION
**Pull request by zaynt4606**
### Why are the changes needed?
There are a small probability of the TestCongestionController test failing.

image

That is because the `checkService` will excute once it was init, which can cause a multithreading conflict with the test code.

image

### What changes were proposed in this pull request?
Fix ut bug.
In fact, `shutDownCheckService` still wont prevent the `checkService` from excuting at once but can make the main testing thread waiting for it to shutDown.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
manual test.

### CODE REVIEW

**Review by onebox-li — APPROVED** [review]
Thanks, LGTM.

**Review by turboFei — APPROVED** [review]

File: worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java [hunk 1]
```diff
@@ -298,6 +299,11 @@ public void close() {
     this.producedBufferStatusHub.clear();
   }
 
+  @VisibleForTesting
+  public void shutDownCheckService() {
+    this.checkService.shutdownNow();
```
Comment (RexXiong) [inline worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java:304]: Use ThreadUtils.shutdown

Comment (zaynt4606) [reply worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java]: Done~

**Review by RexXiong — APPROVED** [review]
LGTM

**Pull request closed without merging**